CHANGELOG-0.9.md

```diff
@@ -0,0 +1,59 @@
+## Changelog since v0.8.0
+
+This release makes Shipper run on management and application clusters
+both, so that Shipper can still enforce the state in case of loss of
+communication between the management and application clusters
+([#272][]).
+
+### Breaking Changes
+
+* All target objects were moved to the application clusters
+  ([#277][]). Shipper will migrate your target objects to the
+  application clusters automatically when it finds them ([#320][]),
+  but it won't clean up the original copies in the management
+  cluster. This is done so that reverting to 0.8 would be painless.
+
+### improvements
+
+* `Shipperctl admin clusters apply` is split into multiple commands,
+  so that each operation can be done separately. For example, this
+  allows operators to only set up the application clusters, without
+  touching the management cluster ([#292][])
+* The fleet summary has been removed from the CapacityTarget because
+  it's not useful now that the target objects are in the application
+  clusters. Instead, we've made the status of the Release object more
+  useful ([#288][])
+
+### Migrating to 0.9
+
+- Run `shipperctl clusters setup management`, `shipperctl clusters
+  join` and `shipperctl clusters setup application` to create the
+  relevant CRDs, service accounts and RBAC objects
+- Make sure your context is set to the management cluster, and apply
+  the Shipper 0.9 management deployment object by doing `kubectl apply
+  -f
+  https://github.com/bookingcom/shipper/releases/download/v0.9.0/shipper-mgmt.deployment.v0.9.0.yaml`
+- Switch your kubectl context to each of the application clusters, and
```

Comment:
I would suggest adding a wait step here to wait for the migration process to finish